documentation-website/Writerside/topics/Get-started-with-Exposed-DAO.md

```diff
@@ -0,0 +1,280 @@
+<include from="Get-Started-with-Exposed.topic" element-id="create-new-project"/>
+
+## Add dependencies
+
+
+<procedure>
+Before you start using Exposed, you need to provide dependencies to your project.
+<step>
+
+Navigate to the **gradle/libs.versions.toml** file and define the Exposed and H2 version and libraries:
+
+```kotlin
+[versions]
+//...
+exposed = "%exposed_version%"
+h2 = "%h2_db_version%"
+
+[libraries]
+//...
+exposed-core = { module = "org.jetbrains.exposed:exposed-core", version.ref = "exposed" }
+exposed-dao = { module = "org.jetbrains.exposed:exposed-dao", version.ref = "exposed" }
+exposed-jdbc = { module = "org.jetbrains.exposed:exposed-jdbc", version.ref = "exposed" }
+h2 = { module = "com.h2database:h2", version.ref = "h2" }
+```
+
+- The `exposed-core` module provides the foundational components and abstractions needed to work with databases in a
+type-safe manner and includes the DSL API.
+- The `exposed-dao` module allows you to work with the Data Access Object (DAO) API.
+- The `exposed-jdbc` module is an extension of the <code>exposed-core</code> module that adds support for Java 
+Database Connectivity (JDBC).
+
+</step>
+<step>
+
+Navigate to the **app/build.gradle.kts** file and add the Exposed and H2 database modules into the `dependencies` block:
+
+```kotlin
+dependencies {
+    //...
+    implementation(libs.exposed.core)
+    implementation(libs.exposed.dao)
+    implementation(libs.exposed.jdbc)
+    implementation(libs.h2)
+    //...
+}
+```
+</step>
+<step>
+    <include from="lib.topic" element-id="intellij_idea_load_gradle_changes"/>
+</step>
+</procedure>
+
+## Configure a database connection
+
+Every database access using Exposed is started by obtaining a connection and creating a transaction.
+To configure the database connection, use the `Database.connect()` function.
+
+<include from="Get-Started-with-Exposed.topic" element-id="config-db-connection-procedure"/>
+
+With this, you've added Exposed to your Kotlin project and configured a database connection.
+You're now ready to define your data model and engage with the database using Exposed's DAO API.
+
+## Define a table object
+
+Exposed's DAO API provides the base `IdTable` class and its subclasses to define tables that use a standard `id` column
+as the primary key.
+To define the table object, follow the steps below.
+
+<procedure id="define-table-object-procedure">
+<step>
+
+In the **app/src/main/kotlin/org/example/** folder, create a new **Task.kt** file.
+
+</step>
+<step>
+
+Open **Task.kt** and add the following table definition:
+
+```kotlin
+```
+{src="get-started-with-exposed-dao/src/main/kotlin/org/example/Task.kt" include-lines="1-2,5,8,10-14"}
+
+In the `IntIdTable` constructor, passing the name `tasks` configures a custom
+name for the table. If you don't provide a name, Exposed will derive it from the object name, which may lead to
+unexpected results depending on naming conventions.
+
+The `Tasks` object defines three columns:
```

Comment:
It's safer to avoid using exact numbers in this context :) to save ourselves some maintenance down the road, when a column is added and there is no easy way to track if we referred to the number of columns anywhere.